src/mercury-agent/mercury/agent/backend_client.py

```diff
@@ -0,0 +1,14 @@
+from mercury.agent.configuration import agent_configuration
+from mercury.common.clients.rpc.backend import BackEndClient
+
+
+# Private
+__backend_client = None
```

Comment:
private is `_`, `__` is for automatic subclass name mangling and is advised against for standalone variables as it can be confused with dundurs.

Reply:
I would take this one step further and say `_` should be avoid here, only to be used to avoid clashing with built-ins. Instead, this file should just have an `__all__` that doesn't include the variable. If someone wants to modify the variable directly, they can no matter what, better to have the comment and call it something like 'private_backend' rather than rely on trying to obscure it.

Reply:
I only do it because it looks pretty.